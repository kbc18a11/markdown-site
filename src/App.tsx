import './App.css';
import { BrowserRouter, Routes, Route } from 'react-router-dom';
import { Header } from 'components/header/Header';
import { Index } from 'pages/Index';
import { initMarkdownTextList } from 'stores/MarkdownTextListStore';

const App = () => {
  initMarkdownTextList();

  return (
    <div className="App">
      <BrowserRouter>
        <Header />
        <Routes>
          <Route
            index
            element={<Index />}
          />
        </Routes>
      </BrowserRouter>
    </div>
  );
};

export default App;
