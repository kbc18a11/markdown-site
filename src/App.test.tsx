import { describe, expect, test } from 'vitest';
import { render } from '@testing-library/react';
import App from './App';

describe('Simple working test', () => {
  test('should render correctly', () => {
    const { container } = render(<App />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
