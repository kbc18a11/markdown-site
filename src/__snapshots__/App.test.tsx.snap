// Vitest Snapshot v1

exports[`Simple working test > should render correctly 1`] = `
<div
  class="App"
>
  <div
    class="MuiBox-root css-i9gxme"
  >
    <header
      class="MuiPaper-root MuiPaper-elevation MuiPaper-elevation4 MuiAppBar-root MuiAppBar-colorPrimary MuiAppBar-positionStatic css-hip9hq-MuiPaper-root-MuiAppBar-root"
    >
      <div
        class="MuiToolbar-root MuiToolbar-gutters MuiToolbar-regular css-1t29gy6-MuiToolbar-root"
      >
        <button
          aria-label="menu"
          class="MuiButtonBase-root MuiIconButton-root MuiIconButton-colorInherit MuiIconButton-edgeStart MuiIconButton-sizeLarge css-ancrnh-MuiButtonBase-root-MuiIconButton-root"
          tabindex="0"
          type="button"
        >
          <svg
            aria-hidden="true"
            class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
            data-testid="MenuIcon"
            focusable="false"
            viewBox="0 0 24 24"
          >
            <path
              d="M3 18h18v-2H3v2zm0-5h18v-2H3v2zm0-7v2h18V6H3z"
            />
          </svg>
          <span
            class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
          />
        </button>
        <div
          class="MuiTypography-root MuiTypography-h6 css-1juivf6-MuiTypography-root"
        >
          Markdonw Site
        </div>
      </div>
    </header>
  </div>
  <div>
    <p>
      A paragraph with 
      <em>
        emphasis
      </em>
       and 
      <strong>
        strong importance
      </strong>
      .
    </p>
    

    <blockquote>
      

      <p>
        A block quote with 
        <del>
          strikethrough
        </del>
         and a URL: 
        <a
          href="https://reactjs.org"
        >
          https://reactjs.org
        </a>
        .
      </p>
      

    </blockquote>
    

    <ul
      class="contains-task-list"
    >
      

      <li>
        Lists
      </li>
      

      <li
        class="task-list-item"
      >
        <input
          disabled=""
          type="checkbox"
        />
         
        todo
      </li>
      

      <li
        class="task-list-item"
      >
        <input
          checked=""
          disabled=""
          type="checkbox"
        />
         
        done
      </li>
      

    </ul>
    

    <p>
      A table:
    </p>
    

    <table>
      <thead>
        <tr>
          <th>
            a
          </th>
          <th>
            b
          </th>
        </tr>
      </thead>
    </table>
  </div>
</div>
`;
